spec/unit/verifier_spec.rb

```diff
@@ -167,6 +167,22 @@
     expect(result.errors.first).to eq "Expected file name: awesomelib.2013-09-08.manifest"
   end
 
+  it "detects invalid topics" do
+    handler = ManifestHandler.new settings
+    handler.read_remote
+    verifier = Verifier.new settings
+
+    manifest = handler.manifest("awesomelib")
+    expect(verifier.verify(manifest).valid?).to be true
+
+    manifest.topics = ["Invalid"]
+
+    result = verifier.verify(manifest)
+
+    expect(result.valid?).to be false
+    expect(result.errors).to include "Invalid topics"
```

Comment:
It would be better to have a more specific error message, such as: "Invalid topic `Invalid`. Valid topics are ...".